src/index.html

```diff
@@ -10,7 +10,28 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class ="search-form search-form_big" data-qa="big">
+      <label class ="search-form__label search-form__label_big">
+        <input
+          data-qa ="keypress"
+          class= "search-form__input  search-form__input_big"
+          type ="text"
+          placeholder ="Try &quot;Los Angeles&quot;"
+        >
+      </label>
+    </form>
+
+    <form class ="search-form search-form_small" data-qa="small">
+      <label class ="search-form__label search-form__label_small">
+        <input
+          data-qa ="hover"
```

Comment:
The data-qa attribute for the small input should be `data-qa="small`

![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/30219274/a49b7d3e-f982-4dc3-b270-74c45c2b4631)